crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -140,15 +176,22 @@ impl SequencerBlockData {
     pub fn from_tendermint_block(b: Block) -> eyre::Result<Self> {
         use astria_sequencer::transaction::Signed;
 
-        if b.header.data_hash.is_none() {
+        let Some(data_hash) = b.header.data_hash else {
             bail!(Error::MissingDataHash);
+        };
+
+        if b.data.is_empty() {
+            bail!("block has no transactions; ie action tree root is missing");
         }
 
+        let action_tree_root =
+            Hash::try_from(b.data[0].clone()).wrap_err("failed to parse action tree root")?;
+
         // we unwrap sequencer txs into rollup-specific data here,
         // and namespace them correspondingly
         let mut rollup_data = HashMap::new();
 
-        for (index, tx) in b.data.iter().enumerate() {
+        for (index, tx) in b.data[1..].iter().enumerate() {
```

Comment:
Can you add a note here why the first one is skipped? Maybe just reference the construction?